plugins/janus_audiobridge.c

```diff
@@ -6740,6 +6820,101 @@ static void *janus_audiobridge_handler(void *data) {
 	return NULL;
 }
 
+static void *janus_audiobridge_rec_add_wav_header(janus_audiobridge_room *audiobridge) {
+
```

Comment:
This empty line is unneeded.

Reply:
@lminiero in most of the code i see empty line between end of function and start of next function. sometimes it is missing. What is the exact convention

Reply:
Fair point! think in the majority of cases it's no-empty line at the beginning, at least in the code I write myself.